modules/zenith-public/lua/2-base/dragoon_adjustments.lua

```diff
@@ -0,0 +1,62 @@
+-- -- -----------------------------------
+-- -- Dragoon Adjustments Module
+-- --
+-- -- This module overrides the base behaviour of Dragoon job abilities.
+-- -- -----------------------------------
+
+require('modules/module_utils')
+
+local m = Module:new('dragoon_adjustments')
+
+local numLevelUps = 0
+local wyvernBonusDA = 0
+
+local wyvernMods = {
+    xi.mod.ACC, 1,
+    xi.mod.HPP, 3,
+    xi.mod.ATTP, 1,
+}
+
+local playerMods = {
+    xi.mod.ATTP, 2,
+    xi.mod.DEFP, 2,
+    xi.mod.HASTE_ABILITY, 100,
+    xi.mod.DOUBLE_ATTACK, wyvernBonusDA / 2,
+    xi.mod.ALL_WSDMG_ALL_HITS, 2,
+}
```

Comment:
```suggestion
local wyvernMods = {
    { xi.mod.ACC, 1 },
    { xi.mod.HPP, 3 },
    { xi.mod.ATTP, 1 },
}

local playerMods = {
    { xi.mod.ATTP, 2 },
    { xi.mod.DEFP, 2 },
    { xi.mod.HASTE_ABILITY, 100 },
    { xi.mod.ALL_WSDMG_ALL_HITS, 2 },
}
```

Note i've removed the DA mod as it can be handled differently, mentioned in another note

Reply:
Changed to suggested